code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -39,6 +43,8 @@
 	name = "riot shotgun"
 	desc = "A sturdy shotgun with a longer magazine and a fixed tactical stock designed for non-lethal riot control."
 	icon_state = "riotshotgun"
+	item_state = "shotgun"
+	fire_delay = 7
```

Comment:
All fire delay does, as i recall, is affect burstfire. The var you wanna mess with is fire_rate

Reply:
This is ported from tg and I've tested it there and it works as expected

Reply:
regardless, please use the fire_rate var to get it to the delay you want instead of this, for homogenization purposes